trade-finance-manager-api/src/v1/__mocks__/amendmentVariables.js

```diff
@@ -0,0 +1,180 @@
+const CONSTANTS = require('../../constants');
+
+const approvedWithoutConditionsBothAmendments = {
+  user: {
+    firstname: 'Bob',
+    surname: 'Smith',
+    email: 'test@test.com',
+  },
+  dealSnapshot: {
+    ukefDealId: '3333',
```

Comment:
Just a minor suggestion when repeating numbers like facility ID and deal ID which will remain the same, nice to declare them as a constant so easier to manage for future. Not a blocker.